src/utils/user-helpers.ts

```diff
@@ -106,3 +107,24 @@ export const getUsername = (person?: CristinPerson | null, topOrgCristinId?: str
 
   return `${personCristinIdentifier}@${topOrgCristinIdentifier}`;
 };
+
+export const userCanDeleteMessage = (user: User, message: Message, ticketType: TicketType): boolean => {
+  const userIsSender = user.nvaUsername === message.sender;
+
```

Comment:
Om man alltid skal ha lov til å slette egne meldinger kan vi kanskje sjekke for det først, og slippe å komplisere alle andre sjekker med det?

```suggestion
if (userIsSender) {
  return true;
}
```